rest-api/src/main/resources/definitions/reauthenticate_request.yml

```diff
@@ -0,0 +1,25 @@
+type: object
```

Comment:
I think I'm going to wrap this into the signIn object and rename it AuthCredentials. For the rest client, saving a single type of object with the credentials for the 3 different pathways to authenticate is a lot easier than having credential types for each pathway.